docs/src/frontend/index.md

```diff
@@ -1126,12 +1126,19 @@ Parses a signed `{!rsh} Int` into a JavaScript number.
 Example:
 
 ```js
-load: /examples/programmer-art/src/index.mjs
-md5: 0779cd5bfb7d45de6a7f46b9cd6c7164
-range: 394 - 402
+import { loadStdlib } from "@reach-sh/stdlib";
```

Comment:
I think you should add this as a "test-case" to `/examples/js-tests` and then cite the lines

Reply:
This is what `frontend/#js_parseInt` looks like after [062ef7c](https://github.com/reach-sh/reach-lang/pull/1115/commits/062ef7c27079f984589ac0cfc4616119f45e1972).

![image](https://user-images.githubusercontent.com/43425812/173881005-ae613d1a-349b-4737-9b07-6933b12c8b33.png)

Reply:
This is what `frontend/#js_parseInt` looks like after [f16706b](https://github.com/reach-sh/reach-lang/pull/1115/commits/f16706becb3514660854cfe52cb417d072cdc271).

![image](https://user-images.githubusercontent.com/43425812/173885776-fb3d1427-9516-42fa-9d31-4e8793f17b51.png)